lib/commanded/event/handler.ex

```diff
@@ -820,6 +820,12 @@ defmodule Commanded.Event.Handler do
     {:stop, reason, state}
   end
 
+  @doc false
+  @impl GenServer
+  def handle_info({:EXIT, _pid, reason}, state) do
+    {:stop, reason, state}
+  end
```

Comment:
```suggestion
  def handle_info({:EXIT, _pid, reason}, state) do
    Logger.debug(fn -> describe(state) <> " subscription EXIT due to: #{inspect(reason)}" end)
    {:stop, reason, state}
  end
```

Reply:
Good catch :) But it's already logged as part of the `terminate/2` callback